src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -315,17 +309,6 @@ mocha.describe('manage_nsfs cli', function() {
             }
         });
 
-        mocha.it('cli bucket update owner email', async function() {
```

Comment:
can you keep it and just replace the email with owner? 
I see I added here a validation for the creation date not getting changed on update

Reply:
Sure, thanks, I changed it to owner update instead of email.
If you want we can create a test for the validation of the creation date not getting changed on update so it would be clear.